### PR TITLE
Changed python health check to use shared .proto file

### PR DESCRIPTION
**Pull request by kpayson64**
We want to remove duplicate proto files from the python directory. #6363, #6060

In order to do that, we need logic to copy over the proto files as part of the python packaging because the proto files need to be bundled in the package.

I've tested the following with
```python setup.py sdist``` and ```python setup.py bdist```

### CODE REVIEW

File: src/python/grpcio_health_checking/commands.py [hunk 1]
```diff
@@ -33,11 +33,17 @@
...
 import subprocess
 import sys
 
 import setuptools
 from setuptools.command import build_py
+from setuptools.command import sdist
+
+ROOT_DIR = os.path.abspath(os.path.dirname(os.path.abspath(__file__)))
+HEALTH_PROTO = os.path.abspath(
+    ROOT_DIR + '../../../proto/grpc/health/v1/health.proto')
```
Comment (soltanmm) [inline src/python/grpcio_health_checking/commands.py]: nit: use `os.path.join(ROOT_DIR, '../../blahblah')` (trailing `/`s aren't a documented result of the `os.path.*` API).

I have a feeling, too, that if you `import pdb; pdb.set_trace()` here and `print ROOT_DIR + '../../../proto/grpc/health/v1/health.proto'` you'll see something surprising.

**Comment by kpayson64** [comment]
Addressed feedback.

File: src/python/grpcio_health_checking/commands.py [hunk 2]
```diff
@@ -43,7 +43,7 @@
 
 ROOT_DIR = os.path.abspath(os.path.dirname(os.path.abspath(__file__)))
 HEALTH_PROTO = os.path.abspath(
-    ROOT_DIR + '../../../proto/grpc/health/v1/health.proto')
+    os.path.join(ROOT_DIR, '../../../proto/grpc/health/v1/health.proto'))
```
Comment (soltanmm) [inline src/python/grpcio_health_checking/commands.py]: This ends up with `HEALTH_PROTO = $REPO_ROOT/proto/grpc/health/v1/health.proto`; one too many up-dirs.

File: src/python/grpcio_health_checking/commands.py [hunk 3]
```diff
@@ -33,11 +33,17 @@
...
 import subprocess
 import sys
 
 import setuptools
 from setuptools.command import build_py
+from setuptools.command import sdist
+
+ROOT_DIR = os.path.abspath(os.path.dirname(os.path.abspath(__file__)))
+HEALTH_PROTO = os.path.abspath(
+    os.path.join(ROOT_DIR, '../../../proto/grpc/health/v1/health.proto'))
```
Comment (soltanmm) [inline src/python/grpcio_health_checking/commands.py]: <sub>GitHub is eating my comments, it seems...</sub>
This ends up with `HEALTH_PROTO = $REPO_ROOT/proto/grpc/health/v1/health.proto`; one too many up-dirs.

**Comment by kpayson64** [comment]
Addressed comments.

**Comment by kpayson64** [comment]
This pull request has been superseded by #6439

**Pull request closed without merging**